### DISCUSSION TITLE
Lovely community I need your help right now more than ever.

### DISCUSSION THREAD

**Discussion by Kaller-LTD** [ohmyzsh/ohmyzsh#10827] [Help]
I have tried everything there is and have looked on every forum. I have been playing around with coding and have been installing repositories from GitLab to name a few. Long story short I am not a developer and experiencing an issue with the terminal now. 

any command shows up as 
`zsh: command not found: `

i tried changing the $PATH no luck either. 

XXX@XX-MacBook-Pro-2 ~ % $PATH 
zsh: no such file or directory: /usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin. 

from what I gather the usr folder is in the bit and the bin is empty at the moment. Is there a way to restart the mac, in order to get a properly functioning mac again?  

plz & thank you in advance

**Comment by baysideengineer** [comment]
Hello @Kaller-LTD, a restart should always be the first step to take. If your problem persists after a restart then you can have a look at [this](https://stackoverflow.com/questions/18428374/commands-not-found-on-zsh) or [this](https://apple.stackexchange.com/questions/422109/how-to-fix-zsh-command-not-found-on-all-command).

**Reply by Kaller-LTD** [reply]
I have tried restarting it. my next question is, will I have the same problem if I restart mac to factory settings?

@saschazengler

**Reply by mcornella** [reply]
> I have tried restarting it. my next question is, will I have the same problem if I restart mac to factory settings?

That's the last thing to try. There are a lot of things you can try before that. See [my reply](https://github.com/ohmyzsh/ohmyzsh/discussions/10827#discussioncomment-2518847).

**Comment by mcornella** [comment]
When you say every command, do you mean nothing works? A simple `ls` command should work, seeing that the value of `$PATH` is `/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin`.

If `ls` works, what are examples of commands that don't work?
